website/docs/about/accessibility-statement.md

```diff
@@ -47,14 +47,14 @@ If we are unable to provide conformance through the component itself, we will pr
 
 ## Assessment approach
 
-The HashiCorp Design Systems Team self-assesses the accessibility of the HashiCorp Design System by the following approaches:
+The Design Systems Team self-assesses the accessibility of Helios by the following approaches:
 
 - Manual testing with assistive technologies
 - Use of available automated checks
 
 ## Feedback
 
-We welcome your feedback on the accessibility of the HashiCorp Design System. Please let us know if you encounter accessibility barriers on the HashiCorp Design System:
+We welcome your feedback on the accessibility of Helios, please let us know if you encounter accessibility barriers in the design system:
```

Comment:
Grammatically, the period was more accurate, as this updated version turns this into a run-on sentence. Alternatively, you can use a semi-colon. 

```suggestion
We welcome your feedback on the accessibility of Helios. Please let us know if you encounter accessibility barriers in the design system:
```